docs/04_User_Guides/autofix_resources.md

```diff
@@ -0,0 +1,55 @@
+[TOC]
+
+## Description, how to apply autoformatting frontend resources
+### What is this?
+In this guide we provide some simple examples for the usage of autofixing functionality.
+Crafty comes with some default linter configuration including among other 
+
+- eslint (js)
+- post-css (css)
+- prettier (js + css)
```

Comment:
These project names are written respectively :
ESLint
PostCSS
Prettier

https://github.com/swissquote/crafty/blob/master/.spelling

Reply:
Also, please write "JavaScript" and "CSS"

Reply:
corrected spelling